pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -560,7 +560,7 @@ def select_precision_plugin(self) -> PrecisionPlugin:
             return PrecisionPlugin()
         if self.precision == 64:
             return DoublePrecisionPlugin()
-        if self.precision == 16:
+        if self.precision in (16, "bfloat16"):
```

Comment:
Should we have a check higher for if self.precision not in (16, 'bfloat16', 32, 64): raise MisConfiguration ?

Reply:
It's a good idea, surprised we do not have this setup already. I'll add in a following PR this :)

Reply:
I think adding a `_validate_precision_type` in the `AcceleratorConnector` would be the right place